### PR TITLE
feat: Release Notes for 0.14

### PR DESCRIPTION
**Pull request by deboer-tim**
### What does this PR do?

Initial release notes for 0.14, primarily Kind based.

### Screenshot/screencast of this PR

Screenshot 2023-04-13 at 8 32 14 AM

### What issues does this PR fix or reference?

Issue #1965.

### CODE REVIEW

File: website/blog/2023-04-14-release-0.14.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+<!--truncate-->
+_________________
+
+## Release details
+
+### Kind Installation
+
+The Kind extensions is now bundled as part of Podman Desktop [1421](https://github.com/containers/podman-desktop/issues/1421)
+and allows you to easily install Kind directly from the status bar [1257](https://github.com/containers/podman-desktop/issues/1257).
```
Comment (themr0c) [inline website/blog/2023-04-14-release-0.14.md]:
```suggestion
and allows you to easily [install Kind directly from the status bar](https://podman-desktop.io/docs/kubernetes/kind/installing-kind) [1257](https://github.com/containers/podman-desktop/issues/1257).
```

File: website/blog/2023-04-14-release-0.14.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+The install is available from the system shell [1516](https://github.com/containers/podman-desktop/issues/1516).
+
+### Kind Clusters in Resources
+
+Create Kind clusters from an updated Settings > Resources page that also allows you to start/stop [1953](https://github.com/containers/podman-desktop/issues/1953) and delete [1977](https://github.com/containers/podman-desktop/issues/1977) them.
```
Comment (themr0c) [inline website/blog/2023-04-14-release-0.14.md]:
```suggestion
[Create Kind clusters from an updated **Settings > Resources** page](https://podman-desktop.io/docs/kubernetes/kind/creating-a-kind-cluster) that also allows you to start/stop [1953](https://github.com/containers/podman-desktop/issues/1953) and delete [1977](https://github.com/containers/podman-desktop/issues/1977) them.
```

File: website/blog/2023-04-14-release-0.14.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+controller (Contour) [1675](https://github.com/containers/podman-desktop/issues/1675).
+
+https://user-images.githubusercontent.com/695993/230328543-71a6d2a4-b6ac-44dd-9219-d0a864dd44f9.gif
+839)
+
+### UI and UX improvements
+
+#### Updated Preferences
+
+The Settings > Preferences page has been updated with a new design [1913](https://github.com/containers/podman-desktop/pull/1913),
```
Comment (themr0c) [inline website/blog/2023-04-14-release-0.14.md]:
```suggestion
The **Settings > Preferences** page has been updated with a new design [1913](https://github.com/containers/podman-desktop/pull/1913),
```

**Review by themr0c — APPROVED** [review]

**Review by cdrage — APPROVED** [review]

**Comment by benoitf** [comment]
@cdrage why did you merge this PR ?
blog post goes live by doing that

**Comment by cdrage** [comment]
> @cdrage why did you merge this PR ? blog post goes live by doing that

I apologize, it should of been just a review, not a merge. https://github.com/containers/podman-desktop/pull/2082 created to revert and then we'll redeploy the blog later.

**Pull request merged**